submissions/dafen173/pop-up2/style.css

```diff
@@ -0,0 +1,180 @@
+body {
+	font-size: 16px;
+	font-family: "Montserrat", "Arial", sans-serif;
+	font-size: 16px;
+}
+
+.nav {
+	display: block;
+	height: auto;
+}
+
+.menu {
+	margin-right: 20px;
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+	align-items: center;
+}
+
+.menuItem {
+	margin-left: 20px;
+}
+
+.menu img,
+.button {
+	width: 24px;
+	height: 24px;
+}
+
+.triangle {
+	display: none;
+	position: absolute;
+	top: 49px;
+	right: 116px;
+	width: 0;
+	height: 0;
+	z-index: 2;
+	border-left: 12px solid transparent;
+	border-right: 12px solid transparent;
+	border-bottom: 12px solid #d4d4d4;
+}
+
+.empty {
+	position: absolute;
+	top: 2px;
+	left: -10px;
+	width: 0;
+	height: 0;
+	border-left: 10px solid transparent;
+	border-right: 10px solid transparent;
+	border-bottom: 10px solid #fff;
+}
+
+.popUp,
+.MoreApps {
+	position: absolute;
+	clip: rect(0 0 0 0);
+}
+
+.button {
+	margin-left: 20px;
+	background-image: url(icons/dots.png);
+	background-size: cover;
+	cursor: pointer;
+}
+
+.menu:hover {
+	cursor: pointer;
+}
+
+.menuItem:hover,
+.button:hover {
+	text-decoration: underline;
+	opacity: 0.5;
+}
+
+.iconsContainer {
+	display: none;
+	position: absolute;
+	width: 280px;
+	height: 362px;
+	top: 60px;
+	border: 1px solid #d4d4d4;
+	border-radius: 2px;
+	box-shadow: 0 2px 10px rgba(0, 0, 0, 0.7);
+	overflow-y: auto;
+}
+
+.popupIcons, 
+.hiddenBlock {
+	margin-top: 10px;
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: center;
+	padding: 0;
+}
+
+.popupIconsItem {
+	display: flex;
+	/*justify-content: start;*/
+	width: 70px;
+	margin-top: 10px;
+	margin-bottom: 10px;
+	padding: 5px 5px 0 5px;
+}
+
+.popupIconsItem a:hover {
+	outline: 1px solid #a1a1a1;
+}
+
+.popupIcons a,
+.hiddenBlock a {
+	text-decoration: none;
+	color: black;
+	padding: 2px;
+	width: 100%;
+	height: 100%;
+}
+
+.popupIconsItem a:focus {
+	outline: 1px solid #3f56fb;
+}
+
+.popupIconsItem img,
+.hiddenBlock img {
+	display: block;
+	width: 48px;
+	height: auto;
+	margin-right: auto;
+	margin-left: auto;
+}
+
+.captions {
+	margin-top: 5px;
+	text-align: center;
+}
+
+.popUp:checked ~ .iconsContainer {
+	display: block;
+}
+
+.popUp:checked ~ .triangle {
+	display: block;
+}
+
+.MorePopUp {
+	display: block;
+	margin-top: 10px;
+	line-height: 35px;
+	text-align: center;
+	width: 280px;
+	background-color: #eee;
+	font-size: 16px;
+}
+
+.hiddenBlock,
+.line {
+	display: none;
+}
+
+.line {
+	color: #eee;
+	width: 200px;
+	opacity: 0.5;
+
```

Comment:
could you please remove an empty line

Reply:
Thanks for the review!
I will correct

Reply:
> could you please remove an empty line

seems to have corrected all the points you indicated
it's okay now?